doc/release-notes/release-notes-5.5.0-rc1.md

```diff
@@ -0,0 +1,407 @@
+  no effect and must be set to a dummy value (0 or null).
+
+Changes to Transaction Fee Selection
+------------------------------------
+
+- The `-mintxfee` and `-sendfreetransactions` options have been removed. These
+  options used to instruct the wallet's legacy transaction creation APIs
+  (`sendtoaddress`, `sendmany`, and `fundrawtransaction`) to increase fees to
+  this limit and to use a zero fee for "small" transactions that spend "old"
+  inputs, respectively. They will now cause a warning on node startup if used.
+
+Changes to Block Template Construction
+--------------------------------------
+
+We now use a new block template construction algorithm documented in
+[ZIP 317](https://zips.z.cash/zip-0317#recommended-algorithm-for-block-template-construction).
+
+- This algorithm no longer favours transactions that were previously considered
+  "high priority" because they spent older inputs. The `-blockprioritysize` config
+  option, which configured the portion of the block reserved for these transactions,
+  has been removed and will now cause a warning if used.
+- The `-blockminsize` option, which configured the size of a portion of the block
+  to be filled regardless of transaction fees or priority, has also been removed
+  and will cause a warning if used.
+- A `-blockunpaidactionlimit` option has been added to control the limit on
+  "unpaid actions" that will be accepted in a block for transactions paying less
+  than the ZIP 317 fee. This defaults to 50.
+
+Removal of Priority Estimation
+------------------------------
+
+- Estimation of "priority" needed for a transaction to be included within a target
+  number of blocks, and the associated `estimatepriority` RPC call, have been
+  removed. The format for `fee_estimates.dat` has also changed to no longer save
+  these priority estimates. It will automatically be converted to the new format
+  which is not readable by prior versions of the software. The `-txconfirmtarget`
+  config option is now obsolete and has also been removed. It will cause a
+  warning if used.
+
+[Deprecations](https://zcash.github.io/zcash/user/deprecation.html)
+--------------
+
+The following features have been deprecated, but remain available by default.
+These features may be disabled by setting `-allowdeprecated=none`. 18 weeks
+after this release, these features will be disabled by default and the following
+flags to `-allowdeprecated` will be required to permit their continued use:
+
+- `deprecationinfo_deprecationheight`: The `deprecationheight` field of
+  `getdeprecationinfo` has been deprecated and replaced by the `end_of_service`
+  object.
+
+Changelog
+=========
+
+ANISH M (3):
+      use SOURCES_PATH instead of local git DEPENDS_SOURCES_DIR
+      report the git-derived version in metrics screen
+      Update src/metrics.cpp by removing v prefix.
+
+Alex Morcos (14):
+      Refactor CreateNewBlock to be a method of the BlockAssembler class
```

Comment:
What the heck is going on with the changelog here?

Reply:
That looks the same it always has. There is time we need to spend on this part of the release script, to improve the layout.

Reply:
I sampled a few commit messages here and the attributions seem correct? They should be pulling from the `Author` field of the commit. I definitely want to refresh the historic release notes once we've improved this data extractor.

Reply:
I don't understand. This commit was authored by Alex Morcos and backported in #6313 from what I can tell.

Reply:
Hm, maybe this is just me not being able to see what's going on properly from my phone.

Reply:
The generated Markdown is broken, because the line breaks aren't sufficient to create a new list item. This is a longstanding problem.